src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,17 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bucketVolume;
+    private boolean track;
```

Comment:
You should add getters and setters

Reply:
I added getters, but setters I never use, I use constructor.

Reply:
Ok